src/test/java/reposense/system/CommandRunnerTest.java

```diff
@@ -215,4 +223,22 @@ public void getShortlogSummary_dateOutOfRange_emptyResult() {
 
         Assert.assertTrue(result.isEmpty());
     }
+
+    @Test
+    public void getCurrentBranch_masterBranch_success() {
+        String currentBranch = CommandRunner.getCurrentBranch(config.getRepoRoot());
+        Assert.assertEquals(currentBranch, "master");
+    }
+
+    @Test
+    public void getCurrentBranch_uncommonDefaultBranch_success() throws GitDownloaderException,
+            InvalidLocationException {
+        uncommonDefaultConfig = new RepoConfiguration(TEST_REPO_UNCOMMON_DEFAULT_GIT_LOCATION,
+                RepoConfiguration.DEFAULT_BRANCH);
+        uncommonDefaultConfig.setFormats(ArgsParser.DEFAULT_FORMATS);
+
+        GitDownloader.downloadRepo(uncommonDefaultConfig);
+        String currentBranch = CommandRunner.getCurrentBranch(uncommonDefaultConfig.getRepoRoot());
+        Assert.assertEquals(currentBranch, "uncommon");
```

Comment:
For `assertEquals`, the convention is that the 1st value should be the 'expected' value, followed by the 'actual' value. This is to make it easier for others to know which is which, especially when debugging the failed tests in the future.

So this 2 should be swapped
```suggestion
        Assert.assertEquals("uncommon", currentBranch);
```